docs/modules/ROOT/pages/index.adoc

```diff
@@ -276,8 +276,11 @@ When you write stages, you have access to both global variables (defined without
 | odsSharedLibVersion
 | ODS Jenkins shared library version, taken from reference in Jenkinsfile.
 
+| bitbucketUrl
+| BitBucket URL - value taken from BITBUCKET_URL. If BITBUCKET_URL is not present, it will default to https://<BITBUCKET_HOST>.
+
 | bitbucketHost
-| BitBucket host - value taken from BITBUCKET_HOST.
+| BitBucket host - value taken from bitbucketUrl.
```

Comment:
Again `BitBucket host - value taken from bitbucketUrl.` I guess there is a mix of Host and URL in the sentence

Reply:
No, this value is really derived from `bitbucketUrl`. I can make the wording clearer.